### PR TITLE
[lunasvg] Add new port

### LINKED ISSUE #15432: [New Port Request] lunasvg
Library name: lunasvg

Library description: a standalone c++ library to create, animate, manipulate and render SVG files.

Source repository URL: https://github.com/sammycage/lunasvg

### PR DESCRIPTION
**Pull request by Pospelove**
closes #15432

### CODE REVIEW

**Comment by Pospelove** [comment]
@JonLiu1993 We probably should accept accept as it is and update the port in other PR. This bugfix doesn't seem essential. And the last released version of the lib is still 2.3.1

**Comment by Pospelove** [comment]
@JonLiu1993 Or I can use latest commit and call version 2.3.1-1 in the vcpkg.json file. Is it an option?

**Comment by JonLiu1993** [comment]
> @JonLiu1993 Or I can use latest commit and call version 2.3.1-1 in the vcpkg.json file. Is it an option?

If Upstream doesn't explicitly state that uwp is not supported, we can't arbitrarily add the supports field unless there is favorable evidence

**Comment by Pospelove** [comment]
@JonLiu1993 Sorry? I guess it was another my PR with supports field changed 😄

**Comment by Pospelove** [comment]
Oops. I see. It's a typo actually.

**Review by JonLiu1993 — APPROVED** [review]

File: ports/lunasvg/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lunasvg",
+  "version": "2.3.1",
+  "description": "lunasvg is a standalone SVG rendering library in C++",
+  "homepage": "https://github.com/sammycage/lunasvg",
+  "license": "MIT",
+  "supports": "!uwp",
```
[the remark this answers is not in the document]
Comment (Pospelove) [reply ports/lunasvg/vcpkg.json]: My mistake! it supports actually

[the remark this answers is not in the document]
Comment (Pospelove) [reply ports/lunasvg/vcpkg.json]: Fixing

**Comment by Pospelove** [comment]
@JonLiu1993 🥺  🙏

**Comment by strega-nil-ms** [comment]
Thanks!

**Comment by spaxio** [comment]
Thank you for your work!

**Pull request merged**